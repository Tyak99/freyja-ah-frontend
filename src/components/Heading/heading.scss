$color: #212529;

.headsi {
  color: color;
  font-size: 18px;
  margin-bottom: 3%;
  margin-top: 2%;
  text-align: center;
}
