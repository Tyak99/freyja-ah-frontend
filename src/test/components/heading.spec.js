import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Heading from '../../components/heading/heading';

const setUp = (props = {}) => {
  const component = shallow(<Heading {...props} title="create an account" />);
  return component;
};
describe('Footer component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
  it('should render children', () => {
    const wrapper = component.find('h2');
    expect(wrapper.find('.headsi').exists()).toBe(true);
  });
});
