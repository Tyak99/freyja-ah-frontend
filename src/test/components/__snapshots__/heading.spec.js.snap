// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component should render without errors 1`] = `
<h2
  className="headsi"
>
  create an account
</h2>
`;
